src/views/ContinuationOut.vue

```diff
@@ -397,10 +397,17 @@ export default class ContinuationOut extends Mixins(CommonMixin, DateMixin,
     return 'Continuation Out'
   }
 
+  /** The message to be displayed in the Certify Component. */
+  get certifyMessage (): string {
+    return ('Note: It is an offence to make or assist in making a false or' +
+      ' misleading statement in a record filed under the Business Corporations Act.' +
+      ' A person who commits this offence is subject to a maximum fine of $5,000.')
+  }
```

Comment:
Can we get this from a resource file, same as before?

```
:message="certifyText(FilingCodes.ANNUAL_REPORT_OT)" <<< change the code obviously
```

Reply:
Sev, regarding this. I added the text to the `business-config.ts` file according to the supported continuation out entities. I had a question though.
![supportted corrections](https://github.com/bcgov/business-filings-ui/assets/122301442/88ceca4e-ed29-426c-af22-7e5e7eb61eeb)

CCCs are Community Contribution Companies? We're going to allow continuation out for those? In the code, the CorpType Enum sets `CCC_CONTINUE_IN` as `CCC`. That's different from what the feature flag sets.

Reply:
Good catch! I have fixed all the feature flags that incorrectly used CCC for those companies (the code is CC).

I'm a bit surprised we didn't notice earlier that Community Contribution Companies were not allowed to do Continuation Out, Consent for Continuation Out, Restoration, Alteration or Correction filings...?

Reply:
^^ Can you please do a bit of testing in Dev (or locally) to verify that Community Contribution Companies are allowed to do the above filings?

Reply:
Confirmed:
![Screenshot (76)](https://github.com/bcgov/business-filings-ui/assets/122301442/24db529c-0019-4863-917d-c7aac6c09985)

Reply:
👍 👍 👍